lib/autosign/validator.rb

```diff
@@ -15,36 +17,19 @@ module Autosign
   #
   # @return [Autosign::Validator] instance of the Autosign::Validator class
   class Validator
-    def initialize()
-      start_logging()
-      settings() # just run to validate settings
-      setup()
+    def initialize
+      start_logging
+      settings # just run to validate settings
+      setup
       # call name to ensure that the class fails immediately if child classes
       # do not implement it.
-      name()
+      name
     end
 
-    # Name of the validator. This must be implemented by validators which
-    # inherit from the Autosign::Validator class. The name is used to identify
-    # the validator in friendly messages and to determine which configuration
-    # file section settings will be loaded from.
-    #
-    # @example set the name of a child class validator to "example"
-    #    module Autosign
-    #      module Validators
-    #        class Example < Autosign::Validator
```

Comment:
These comments are no longer needed since the user is not required to implement this method anymore.  Only the NAME constant.